libs/shared/src/entities/message-template/message-template.interface.ts

```diff
@@ -59,4 +60,17 @@ export const SystemVariablesWithTypes = {
     logo: 'string',
     color: 'string',
   },
+  tenant: {
+    name: 'string',
+    data: 'object',
+  },
+};
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const TriggerReservedVariables = ['tenant', 'actor'];
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const ReservedVariablesMap = {
+  [TriggerContextTypeEnum.TENANT]: [{ name: 'identifier', type: TemplateVariableTypeEnum.STRING }],
+  [TriggerContextTypeEnum.ACTOR]: [{ name: 'subscriberId', type: TemplateVariableTypeEnum.STRING }],
```

Comment:
The variables we need to be included for our 'reserved' variables to work in the context of a trigger.